client/galaxy/scripts/mvc/collection/list-of-pairs-collection-creator.js

```diff
@@ -568,7 +568,8 @@ var PairedCollectionCreator = Backbone.View.extend( baseMVC.LoggableMixin ).exte
      */
     createList : function( name ){
         var creator = this,
-            url = '/api/histories/' + this.historyId + '/contents/dataset_collections';
+            root = ( window.Galaxy && Galaxy.options.root )? Galaxy.options.root : '/',
```

Comment:
Just out of interest, shouldn't this be offered as a function `get_galaxy_root()`?